imagelab/ImgProvider.java

```diff
@@ -490,5 +497,25 @@ public void showPixNew(String name) {
         dImage1.changeImage(this,"Second Pass");
         System.out.println("ImgProvider:showSlow: Second Pass");
     }
+    
+        public void removeContainer(){
```

Comment:
I have removed this method as it is not used. The change will be in the next PR.